lib/sequencescape-api/version.rb

```diff
@@ -1,5 +1,5 @@
 module Sequencescape
   class Api
-    VERSION = '0.5.0'.freeze
+    VERSION = '0.6.0-rc1'.freeze
```

Comment:
Shouldn't a potentially breaking change be a major version? -> 1.0.0 ?

Reply:
General semver rules only apply after you hit version 1. Before v1 the second digit generally indicates a breaking change. TBH, we should probably just jump to version 1, no real reason to do a perpetual beta.

Reply:
Oh that's odd, didn't know that! Do you know the reason for it?

Reply:
Its mostly to allow version numbers before 1.0 to indicate expected instability in the API, so you can iterate quickly until you've got something that feels solid. Unless you mean its odd that we haven't gone 1.0, in which case there's no particular reason, just never taken the step. And given we're slowly migrating over to API 2.0, it feels a little odd to do it now.